10up/simple-page-ordering
# Update Cypress Utils and related packages.

### Description of the Change


This upgrades Cypress and the 10up Cypress Utilities and refactors the E2E tests to account for compatibility changes.

This also resolves the issues in dependabot's PR #157 


Closes #155

### How to test the Change


1. Check E2E tests are passing on this PR's [checks tab](https://github.com/10up/simple-page-ordering/pull/160/checks).
2. Run the E2E tests locally:
   1. Ensure you are running the node versions specified in the nvmrc file (16)
   2. `npm i; composer i; npm run build;`
   3. `npm run env destroy`
   4. `npm run env:start`
   5. `npm run cypress:run`

### Changelog Entry

> Changed - Updated 10up Cypress Utilities to 0.2.0
> Security - Update `@cypress/request` to 3.0.0 to resolve SSRF issue


### Credits

Props @peterwilsoncc, @iamdharmesh, @dkotter  


### Checklist:
